docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  db:
+    image: postgres:9.6
+    environment:
+      - POSTGRES_DB=simpl
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+  web:
+    build:
+      context: .
+    volumes:
+      - .:/code
+    ports:
+      - "8100:8000"
+    command: ["/bin/bash", "/code/start.sh"]
+    environment:
+      - DATABASE_URL=postgres://postgres@db/postgres
```

Comment:
isn't the database called `simpl` from the `POSTGRES_DB` env var in the `db` service?

Reply:
Good catch!

Reply:
@frankwiles If this was a good catch, why didn't anything change?

Reply:
@frankwiles Did you change anything?